screens/HomeScreen.js

```diff
@@ -25,6 +24,10 @@ export default function HomeScreen() {
 
   const handleSubmitForm = () => {
     if (name !== '' && amount > 0) {
+      console.log('string?', amount);
+      // parseFloat(amount);
```

Comment:
why was your idea to use `parseFloat`? what do you expect from it?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat